main.js

```diff
@@ -0,0 +1,5 @@
+var scheduledDay = document.querySelector("#scheduled-day");
+var week = document.querySelector(".week");
+week.addEventListener('click', function(event){
+    scheduledDay.innerHTML = "Scheduled Events for " + event.target.innerHTML;
+})
```

Comment:
Put this in a named function, your code will get unreadable quickly
and what happens if you click the container?